src/components/Profile/MyPosts/MyPosts.jsx

```diff
@@ -12,13 +12,14 @@ const MyPosts = (props) => {
   const newPostElement = React.createRef();
 
   const creatPost = () => {
-    const text = newPostElement.current.value;
-    props.addPost(text);
+    // const text = newPostElement.current.value;
+    props.dispatch({type: 'ADD-POST'});
```

Comment:
Consider storing `{type: 'ADD-POST'}` in a separate variable called "action" as you did in "onPostChange"

Reply:
done